guide/ops/starting-stopping-monitoring.md

```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
 layout: website-normal
 ---
 
-**NOTE:** This document is for information on starting an Apache Brooklyn
-Server.  For information on using the Brooklyn Client CLI to access an already
-running Brooklyn Server, refer to [Client CLI Reference](cli/index.html).
+**NOTE:** This document is for information on starting an Apache Brooklyn Server.  For information on using the Brooklyn Client CLI to access an already running Brooklyn Server, refer to [Client CLI Reference](cli/index.html).
 
-## Packages for RHEL/CentOS and Ubuntu
+<ul class="nav nav-tabs">
+    <li class="active impl-1-tab"><a data-target="#impl-1, .impl-1-tab" data-toggle="tab" href="#">Vagrant</a></li>
+    <li class="impl-2-tab"><a data-target="#impl-2, .impl-2-tab" data-toggle="tab" href="#">Centos / RHEL</a></li>
+    <li class="impl-3-tab"><a data-target="#impl-3, .impl-3-tab" data-toggle="tab" href="#">Ubuntu / Debian</a></li>
+    <li class="impl-4-tab"><a data-target="#impl-4, .impl-4-tab" data-toggle="tab" href="#">OSX / Linux</a></li>
+</ul>
 
-If you are using the `.rpm` or `.deb` package of Apache Brooklyn, then Brooklyn
-will integrate with your OS service management. Commands such as
-`service brooklyn start` will work as expected, and Brooklyn's PID file will be
-stored in the normal location for your OS, such as `/var/run/brooklyn.pid`.
+<div class="tab-content">
+<strong class="hidden started-pdf-include">Vagrant</strong>
 
+Now start the provided brooklyn instance supplying your brooklyn Download credentials in place of the `myuser` and `mypassword` exbrooklynles below
```

Comment:
Brooklyn doesn't have download credentials.
I presume remove the `user=myuser password=mypassword` below as well.

What is "exbrooklynles"?